applications/GeoMechanicsApplication/tests/cpp_tests/test_axisymmetric_stress_state.cpp

```diff
@@ -0,0 +1,108 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//                   Marjan Fathian
+//
+
+#include "containers/model.h"
+#include "custom_elements/axisymmetric_stress_state.h"
+#include "custom_elements/stress_state_policy.h"
+#include "geometries/geometry.h"
+#include "includes/checks.h"
+#include "testing/testing.h"
+#include <boost/numeric/ublas/assignment.hpp>
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+ModelPart& CreateModelPart(Model& rModel)
+{
+    ModelPart& result = rModel.CreateModelPart("Main");
```

Comment:
I would prefer to use `auto` here:
```suggestion
    auto& result = rModel.CreateModelPart("Main");
```

Reply:
Done